Kentor.AuthServices/SAML2P/Saml2RequesterId.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Xml.Linq;
+
+namespace Kentor.AuthServices.Saml2P
+{
+    /// <summary>
+    /// Saml2RequesterId - represents a requesting entity on whose behalf the requester is acting.
+    /// </summary>
+    public class Saml2RequesterId
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Saml2RequesterId"/> class.
+        /// </summary>
+        /// <param name="id">The identifier.</param>
+        public Saml2RequesterId(Uri id)
+        {
+            Id = id;
+        }
+
+        /// <summary>
+        /// Gets and sets the id.
+        /// </summary>
+        public Uri Id { get; set; }
```

Comment:
Is this an Entity ID? In that case, please use the EntityId class instead of URI.